etc/httpd/global.conf

```diff
@@ -603,7 +603,7 @@ RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?mobile/android-download(-beta)?\.html)$ /
 RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?projects/calendar(/.*)?)$ /b$1 [PT]
 
 # bug 947890
-RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/latest/releasenotes(/?)$ /b/$1firefox/latest/releasenotes$2 [PT]
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/(?:latest/)?releasenotes(/?)$ /b/$1firefox/latest/releasenotes$2 [PT]
```

Comment:
Ah. Gotcha. I just don't want a mismatch to make things strange later. My preference would be to make the `(latest/)?` bit capture and use it in the destination so that Django sees the real request.